### PR TITLE
feat: add link to learn more about nutriscore + ecoscore

### LINKED ISSUE #6689: Add the link to the full Nutri-Score explainer page at the end of the methodology knowledge panel
## What

- [ ] Add the link to the full Nutri-Score explainer page at the end of the methodology knowledge panel
- Similar to https://github.com/openfoodfacts/openfoodfacts-server/blob/main/templates/api/knowledge-panels/health/ingredients/nova.tt.json#L54
- Possibly in https://github.com/openfoodfacts/openfoodfacts-server/blob/main/templates/web/pages/product/includes/nutriscore_details.tt.html

### Screenshots
image
image

### PR DESCRIPTION
**Pull request by stephanegigandet**
fixes #6689

related to #6679

### CODE REVIEW

File: templates/api/knowledge-panels/health/nutriscore/nutriscore.tt.json [hunk 1]
```diff
@@ -31,5 +31,13 @@
                 "html": `[% panel.nutriscore_details %]`,
             }
         },
+        {
+            "element_type": "text",
+            "text_element": {
+                "html": `
+<p>&rarr; <a href="/nutriscore">[% lang('nutriscore_learn_more') %]</a></p>
```
Comment (alexgarel) [inline templates/api/knowledge-panels/health/nutriscore/nutriscore.tt.json:38]: Why not `url_for_text("nutriscore")` here ?

**Review by alexgarel — APPROVED** [review]
LGTM but a style comment !

**Review by teolemon — APPROVED** [review]
What's happening to the existing NOVA link ?

**Comment by stephanegigandet** [comment]
> What's happening to the existing NOVA link ?

Nothing is happening, it's kept as-is. This PR adds a similar "Learn more about... " link in the Nutri-Score and Eco-Score panels.

**Comment by stephanegigandet** [comment]
Can we close this?

**Comment by teolemon** [comment]
@stephanegigandet I can't see it in Smoothie. Has it been deployed ? Do we need tweaking client side ?

**Pull request merged**